docs/source/workflows/design_spaces.rst

```diff
@@ -10,11 +10,11 @@ Currently, there are two design spaces:
 Product design space
 --------------------
 
-Materials from a product design space are composed from the outer product of univariate dimensions.
+Materials from a product design space are composed from the Cartesian product of univariate dimensions.
```

Comment:
Why do you expect "Cartesian product" to be more intuitive to readers than "outer product?"

Reply:
I see that Cartesian product is more correct, but I had to look that up, and outer product was more immediately intuitive to me.

Reply:
I agree that outer product is more well known as a term. However, I think there is a risk of confusion: for both of these products, I'd expect people to intuit "I should get a matrix." 

But:

- For outer product, the elements of the matrix should be products themselves
- For Cartesian product, the elements of the matrix should be coordinates

Compromise: What if we said something like `outer (Cartesian) product`? Then it's hopefully clear precisely what we mean -- coordinates, and not multiplication -- without relying on the reader's knowledge of the less common product.

Reply:
When in doubt I think we should go for correctness. If the user is not familiar with the term "Cartesian product" then they will Google it and immediately get a box with a definition and a clear picture. If we make up/muddle terms then we risk more confusion in the long run.

Reply:
Yes, you guys are right. I realized I'm being hypocritical here having harped so much on "MLI" vs. "LI" - we should be correct first and it's partially our job to teach customers what these concepts mean if they're not intuitive.